pymtl3/passes/GenDAGPass.py

```diff
@@ -164,6 +164,7 @@ def _process_value_constraints( self, top ):
     write_upblks = defaultdict(set)
 
     constraint_objs = defaultdict(set)
+    expl_constraints = set( top._dsl.all_U_U_constraints )
```

Comment:
do you still need this?

Reply:
Got an error when it hits https://github.com/cornell-brg/pymtl3/blob/bd7da096cd0dabd4a3bbc0ddc07931a51728b323/pymtl3/passes/GenDAGPass.py#L188-L202

Reply:
oh right. I haven't touched this part for a while ...